Swift Chat Demo 2/Swift Chat Demo 2.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
I think we will need an production version for release. We usually create 2 entitlements, `development` for `Debug` and `production` for release.

Reply:
I think `requestAuthorization`  has to be called before `registerForRemoteNotifications` ?

Reply:
Yes, I checked the doc again. If we don't use `requestAuthorization`, the notification will become silent. So you are right! thanks @howawong

https://developer.apple.com/documentation/uikit/uiapplication/1623078-registerforremotenotifications